### PR TITLE
 added ky, uz, hy, be, bg, bs, lv, lt, mn, mk, ro, sr, sk, sl, fa, hr,

### PR DESCRIPTION
**Pull request by madianas**
added ky, uz, hy, be, bg, bs, lv, lt, mn, mk, ro, sr, sk, sl, fa, hr

Source: https://github.com/unicode-cldr/cldr-localenames-full/blob/master/main/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,31 +17,50 @@
     "countries",
     "ar",
     "az",
+    "be",
+    "bg",
+    "bs",
     "cs",
     "da",
     "de",
+    "el",
```
Comment (michaelwittig) [inline package.json]: seems to be a duplicate?

File: README.md [hunk 2]
```diff
@@ -44,31 +44,46 @@ console.log(countries.getNames("en")); // { 'AF': 'Afghanistan', 'AL': 'Albania'
...
+* `mk`: Macedonian
+* `mn`: Mongolian
 * `nb`: Norwegian Bokmål
 * `nl`: Dutch
 * `nn`: Norwegian Nynorsk
 * `pl`: Polish
 * `pt`: Portuguese
+* `ro`: Romanian
 * `ru`: Russian
+* `sk`: Slovak
```
Comment (michaelwittig) [inline README.md]: sl is missing here?

**Comment by michaelwittig** [comment]
Hi @madianas that's an awesome contribution! I found two very tiny issues. Could you double check them and IMm very happy to merge this!

File: README.md (hunk 2 above)
Comment (madianas) [reply README.md]: Nope, Slovak is sk. https://en.wikipedia.org/wiki/Slovak_language
sl is for Slovene https://en.wikipedia.org/wiki/Slovene_language

File: package.json (hunk 1 above)
Comment (madianas) [reply package.json]: Yeah, I thought I lost it after sorting alphabetically. I will remove the one below. It's out of order.

File: README.md (hunk 2 above)
Comment (madianas) [reply README.md]: I will fix that, now too :) good catch

**Comment by michaelwittig** [comment]
published as `i18n-iso-countries@3.1.0`

**Pull request merged**